src/main/java/com/github/bedrin/jdbc/sniffer/log/QueryLogger.java

```diff
@@ -1,9 +1,37 @@
 package com.github.bedrin.jdbc.sniffer.log;
 
-public class QueryLogger {
+public abstract class QueryLogger {
+
+    private final static QueryLogger INSTANCE;
+
+    static {
+        INSTANCE = createQueryLogger();
+    }
+
+    static QueryLogger createQueryLogger() {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/sniffy/sniffy/file/1630181943/issues/source?bid=3005960&fileBranchId=3033956#l11)